apps/cicognara-static/_includes/search_footer.html

```diff
@@ -6,18 +6,21 @@
   var runSearch = function() {
     var i, item;
 
-    const fuseOptions = {
-            keys: ["filename",
-               "ciconum",
-               "author",
-               "title",
-               "people",
-               "orgs",
-               "languages",
-               "dates",
-               "places",
-               "keywords",
-              ],
+      const fuseOptions = {
+          ignoreLocation: true,
+          threshold: 0.4,
```

Comment:
My only question is whether we should make this even lower? 0.2 or 0.1 produce results closer to the current site in the handful of searches I tried.

Reply:
Given that our goal is to produce parity with the current site, that sounds perfectly reasonable.